fem/datacollection.hpp

```diff
@@ -457,6 +457,12 @@ class VisItDataCollection : public DataCollection
    virtual void RegisterQField(const std::string& q_field_name,
                                QuadratureFunction *qf);
 
+   /// Set the number of digits used for both the cycle and the MPI rank
+   /// @note VisIt seems to require 6 pad digits for the MPI rank. Therefore,
+   /// this function uses this default value. This behavior can be overridden
+   /// by calling SetPadDigitsCycle() and SetPadDigitsRank() instead.
+   void SetPadDigits(int digits) { pad_digits_cycle=digits; pad_digits_rank=6; }
+
```

Comment:
Right, I forgot to change that.

Reply:
I thought it was best to leave `SetPadDigitsRank` as it is. If someone has a specific reason for altering this then we should let them. The current versions of VisIt may not be able to load the files but perhaps that is not their goal.

Reply:
I can alter `load-dc` and `convert-dc` to support different pad lengths. This should be a quick fix.

Reply:
Have you noticed any other programs that may be impacted?

Reply:
As far as I am aware no other code uses the loading logic (minus the now failing test).

Reply:
Thanks @termi-official.

Marking this as `override` does sound correct and useful but there seems to be no precedent in MFEM. If we do this it would seem wiser to mark all of the overrides in this class at least (or even better all classes in this header file). Creating this type of precedent may require a vote from a wider group of developers. What do you say @tzanio?

Reply:
We discussed this at our weekly PR meeting and it seems I was mistaken, we do use `override` elsewhere in MFEM. I've added the `override` decoration to the relevant member functions of `VisItDataCollection`. Thanks for suggesting this, @termi-official.

Reply:
Thanks for taking care!